odoo/addons/base/models/ir_qweb.py

```diff
@@ -845,6 +888,16 @@ def _prepare_globals(self):
             'Markup': Markup,
             'escape': escape,
             'VOID_ELEMENTS': VOID_ELEMENTS,
+            'QWebException': QWebException,
+            'Exception': Exception,
+            'TransactionRollbackError': TransactionRollbackError, # for SerializationFailure in assets
```

Comment:
I had put a lambda before (to avoid having access to the cursor). Seen with @odony , this does not seem to be a problem, especially since 'env' is one of the values.

Reply:
yes normally it won't be an issue, but it's just to try to give the strict minimum types to the execution context

you can keep it like this if you want